CHANGES.rst

```diff
@@ -6,6 +6,7 @@ Version 2.3.5
 Unreleased
 
 -   Python 3.12 compatibility. :issue:`2704`
+-   Fix handling of invalid base64 values in ``Authorization.from_header`` :issue:`2717`
```

Comment:
Missing a period (`.`) at the end of the sentence (see previous line for example).